src/components/WalletsList/index.js

```diff
@@ -0,0 +1 @@
+export { WalletsList } from './WalletsList';
```

Comment:
I have doubts about this naming.
If you have any better suggestions - welcome!

Reply:
Why do you have doubts? Sounds logic.

Reply:
Agree with Pavel. Maybe remove letter 's' and leave like WalletList?

Reply:
I thought about it, but 'WalletList' would be not 100% descriptive, because we will have in this component info not only about one wallet.

Reply:
As i understood Wallet List == List of Wallets and it doesn't mean it hold only one wallet

Reply:
> As i understood Wallet List == List of Wallets and it doesn't mean it hold only one wallet

for someone WalletList can mean a list of some propeties for wallet, not list of wallets :)